app/views/organisations/_organisations.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="grid-row">
+  <div class="column-one-third">
+    <% unless @organisations.number_10?(organisation_type) %>
+      <h2><%= organisation_type %></h2>
+      <div class="department-count">
+        <%= organisations.count %>
+      </div>
+    <% end %>
+  </div>
+  <div class="column-two-thirds">
+    <div class="organisation">
+      <% organisations.each do |org| %>
+        <h3 class="organisation-title">
+          <%= org["title"] %>
+        </h3>
+        <p class="organisation-description">
+          <% unless @organisations.number_10?(organisation_type) || @organisations.organisation_department_count(org) == 0 %>
+            <%= "Works with #{@organisations.organisation_department_count(org)} agencies and public bodies" %>
+            <%= link_to "View more", "#", class: "organisation-view-more" %>
+          <% end %>
+        </p>
+        <div class="organisation-work-with-details">
+          <div class="organisation-department-categories">
+            <% org["works_with"].each do |category_type, departments| %>
+              <%= render 'organisation_department_category',
+                         category_type: category_type,
+                         departments: departments %>
+            <% end %>
+          </div>
+        </div>
+      <% end %>
+    </div>
+    <hr>
+  </div>
+</div>
```

Comment:
The red symbol here means that the file hasn't been finished with a new line which it should.  Most editors have a setting to ensure that this happens.  I've been got by this - see: https://github.com/alphagov/frontend/pull/907 ;-)